bundles/core/org.eclipse.smarthome.core.test/src/test/java/org/eclipse/smarthome/core/library/items/RollershutterItemTest.java

```diff
@@ -49,4 +57,43 @@ public void setState_stateDecimal050_returnPercent50() {
         assertEquals(new PercentType(50), sut.getState());
     }
 
+    @Test
+    public void setState_stateOpen_returnPercent100() {
+        RollershutterItem sut = new RollershutterItem("Test");
+        State state = OpenClosedType.OPEN;
+        sut.setState(state);
```

Comment:
The accepted data types of a RollershutterItem are only Percent, UpDown and UnDef - I wonder if we should at all allow other types being set and if so, if we really expect a certain outcome? Are these tests based on some real requirement that you came across? If not, I would tend to say that we rather should ignore such calls and log an error to indicate that something is wrong in the system.